roles/StackStorm.st2chatops/tasks/main.yml

```diff
@@ -1,4 +1,18 @@
 ---
+- name: Temporary hack until proper upstream fix https://bugs.centos.org/view.php?id=13669
+  yum:
+     list: http-parser
+  when: ansible_facts.os_family == 'RedHat' and ansible_facts.distribution_major_version == '8'
+  register: http_parser
+  tags: st2chatops
+
+- name: Install http-parser if needed
+  yum:
+      name: http://repo.okay.com.mx/centos/8/x86_64/release//http-parser-2.8.0-2.el8.x86_64.rpm
+      state: present
+  when: ansible_facts.os_family == 'RedHat' and ansible_facts.distribution_major_version == '8' and (http_parser.results|length == 0)
+  tags: st2chatops
+
```

Comment:
```suggestion
```
This could be removed.

Per https://bugs.centos.org/view.php?id=13669 the bug is not related to `el8`, @punkrokk copy-pasted it by mistake when added el8 support.
I'm still waiting him to fix that in https://github.com/StackStorm/st2-packages/blob/29a493e33de928e6df452a9102e80aaeadc5ec53/scripts/st2bootstrap-el8.sh#L694-L697 :wink:

Reply:
Here is the follow-up fix for that: https://github.com/StackStorm/st2-packages/pull/651
cc @punkrokk